internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,67 @@
 package daysteps
 
 import (
-	...
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+	"yp-go5/internal/personaldata"
+	"yp-go5/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
 // создайте структуру DaySteps
-...
-
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
 
 // создайте метод Parse()
-...
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	sData := strings.Split(datastring, ",")
+	if len(sData) != 2 {
+		return errors.New("invalid input data")
+	}
+
+	steps, err := strconv.Atoi(sData[0])
+	if err != nil {
+		return
```

Comment:
Если мы просто выйдем, то вызывающая сторона никак не узнает об ошибки, а нам это не нужно.
Нужно обернуть ошибку, обогатив ее доп инфой
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```